### PR TITLE
Improve docs about helm chart

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind cleanup
/kind documentation

**What this PR does / why we need it**:
1. Based on #1984, we will add a CI workflow for helm chart. And first we will modify helm chart dir. Meanwhile we need update docs.
2. Improve Flux docs which will be posted in Flux community.

**Which issue(s) this PR fixes**:
Fixes # 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

File: docs/working-with-flux.md [hunk 1]
```diff
@@ -102,53 +123,89 @@ spec:
...
 podinfo	default  	1       	2022-05-27 01:44:35.24229175 +0000 UTC	deployed	podinfo-5.0.3	5.0.3
 ```
 
-Also, you can use Karmada's OverridePolicy to customize applications for specific clusters. For example, if you just want to change replicas in member1, you can refer to the overridePolicy below.
+Based on Karmada's propagation policy, you can schedule Helm releases to your desired cluster flexibly, just like Kubernetes scheduling Pods to the desired node.
+
+## Customize the Helm release for specific clusters
+
+The example above shows how to distribute the same Helm release to multiple clusters in Karmada. Besides, you can use Karmada's OverridePolicy to customize applications for specific clusters.
+For example, If you just want to change replicas in member1, you can refer to the overridePolicy below.
```
Comment (Tingtal) [inline docs/working-with-flux.md:161]: Change If to if

File: docs/working-with-flux.md [hunk 2]
```diff
@@ -102,53 +123,89 @@ spec:
...
+```
+
+The output is similar to:
+
+```
+overridepolicy.policy.karmada.io/example-override configured
 ```
 
-After that, you can find that replicas in member1 has changed.
+3. After applying the above policy in Karmada control plane, you will find that replicas in member1 has changed to 2, but those in member2 keep the same.
```
Comment (Tingtal) [inline docs/working-with-flux.md:200]: Should be: you will find that replicas in member1 **have** changed to 2

**Review by Tingtal — COMMENTED** [review]
2 small grammatical changes

**Comment by RainbowMango** [comment]
Thanks @Tingtal .

@Poor12 Please help create a [good-first](https://github.com/karmada-io/karmada/blob/master/.github/ISSUE_TEMPLATE/good-first.md) issue for the comments.

**Pull request merged**